source/set-mutable.d.ts

```diff
@@ -0,0 +1,35 @@
+import {Except} from './except';
+import {Mutable} from './mutable';
+
+/**
+Create a type that converts the given keys from `readonly` to mutable. The remaining keys are kept as is.
+
+Use-case: You want to define a single model where the only thing that changes is whether or not some of the keys are mutable.
+
+@example
+```
+import {SetMutable} from 'type-fest';
+
+type Foo = {
+	readonly a: number;
+	b: string;
+	readonly c: boolean;
+}
+
+type SomeMutable = SetMutable<Foo, 'b' | 'c'>;
+// type SomeMutable = {
+// 	readonly a: number;
+// 	b: string; // Was already mutable and still is.
+// 	c: boolean; // Is now mutable.
+// }
+```
+*/
+export type SetMutable<BaseType, Keys extends keyof BaseType = keyof BaseType> =
```

Comment:
The second typearg has a default of `keyof BaseType`, doesn't this reduce down to the same as `Mutable` if that default is used? Is there a use case for `SetMutable<T>`?

Reply:
The difference between `Mutable` and `SetMutable` is that in the latter you can set only some keys as mutable, while leaving the rest unafffected.

Reply:
There really isn't, but that's the way the other `Set-` types work, so I just replicated that pattern.

Reply:
If there really isn't any use-case for that type parameter, we should remove it from those other types too.